docs/application-connector/docs/08-03-register-manage-services.md

```diff
@@ -128,3 +128,46 @@ A successful response returns the ID of the registered service:
     ```
     curl https://gateway.kyma.local:{NODE_PORT}/{APP_NAME}/v1/metadata/services/{YOUR_SERVICE_ID} --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
     ```
+
+
+## Register API with specification URL
+
+Application Registry allows you to pass API specification in form of specification URL.
+
+To register API with specification URL replace `api.spec` with `api.specificationUrl`. Be aware that `api.spec` has higher priority than `api.specificationUrl`.
```

Comment:
```suggestion
To register API with a specification URL, replace **api.spec** with **api.specificationUrl**. Note that **api.spec** has a higher priority than **api.specificationUrl**.
```

Reply:
``Be aware that `api.spec` has higher priority than `api.specificationUrl`.`` -> so what? Please provide more info.

Reply:
Done!